ncloud/data_source_ncloud_server_product.go

```diff
@@ -18,140 +19,113 @@ func dataSourceNcloudServerProduct() *schema.Resource {
+				Optional: true,
+				Computed: true,
 			},
 			"zone": {
-				Type:        schema.TypeString,
-				Optional:    true,
-				Description: "Zone code. Get available values using the `data ncloud_zones`.",
+				Type:     schema.TypeString,
+				Optional: true,
 			},
 			"internet_line_type_code": {
 				Type:         schema.TypeString,
 				Optional:     true,
 				ValidateFunc: validation.StringInSlice([]string{"PUBLC", "GLBL"}, false),
-				Description:  "Internet line identification code. PUBLC(Public), GLBL(Global). default : PUBLC(Public)",
 			},
 			"filter": dataSourceFiltersSchema(),
 
 			"product_name": {
-				Type:        schema.TypeString,
-				Computed:    true,
-				Description: "Product name",
+				Type:     schema.TypeString,
+				Computed: true,
 			},
 			"product_type": {
-				Type:        schema.TypeString,
-				Computed:    true,
-				Description: "Product type",
+				Type:     schema.TypeString,
+				Computed: true,
 			},
 			"product_description": {
-				Type:        schema.TypeString,
-				Computed:    true,
-				Description: "Product description",
+				Type:     schema.TypeString,
+				Computed: true,
 			},
 			"infra_resource_type": {
-				Type:        schema.TypeString,
-				Computed:    true,
-				Description: "Infra resource type",
-			},
-			"infra_resource_detail_type_code": {
-				Type:        schema.TypeString,
-				Computed:    true,
-				Description: "infra resource detail type code.",
+				Type:     schema.TypeString,
+				Computed: true,
 			},
 			"cpu_count": {
-				Type:        schema.TypeInt,
-				Computed:    true,
-				Description: "CPU count",
+				Type:     schema.TypeInt,
+				Computed: true,
 			},
 			"memory_size": {
-				Type:        schema.TypeInt,
-				Computed:    true,
-				Description: "Memory size",
+				Type:     schema.TypeString,
+				Computed: true,
 			},
 			"base_block_storage_size": {
-				Type:        schema.TypeInt,
-				Computed:    true,
-				Description: "Base block storage size",
-			},
-			"platform_type": {
-				Type:        schema.TypeString,
-				Computed:    true,
-				Description: "Platform type",
-			},
-			"os_information": {
-				Type:        schema.TypeString,
-				Computed:    true,
-				Description: "OS Information",
+				Type:     schema.TypeString,
+				Computed: true,
 			},
 			"disk_type": {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
-			"add_block_storage_size": {
-				Type:        schema.TypeInt,
-				Computed:    true,
-				Description: "Additional block storage size",
-			},
 			"generation_code": {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			// Deprecated
+			"product_name_regex": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.ValidateRegexp,
+				Deprecated:   "use filter instead",
+			},
+			"exclusion_product_code": {
+				Type:       schema.TypeString,
+				Optional:   true,
+				Deprecated: "This field no longer support",
```

Comment:
```suggestion
				Deprecated: "This field is no longer support",
```
`is` 를 넣는게 맞을가요? 다른 부분도 같이 확인 부탁 합니다.